doc/motr-developer-guide.md

```diff
@@ -209,7 +209,67 @@ as the last argument to this program. In this example, we will only use this id
 64-bit of an object identification. This id should be larger than 0x100000ULL, that is 1048576 in decimal.
 
 ## A simple Cortx Motr application (index)
-   * TODO
+Here we will create a simple Cortx Motr application to crate an index, put
+some key/value pairs to this index, read key/value pairs back from this index, and then delete it.
```

Comment:
Style: make some use of pronouns:

"... put some key/value pairs to this index, read them back, and then delete the index."

Reply:
fixed.